docs/docs/guide/quick-start.md

```diff
@@ -14,6 +14,20 @@ DXOS is the developer platform for **collaborative**, **offline-first**, **priva
 *   [Installing KUBE infrastructure](#starting-a-kube) for hosting apps locally
 *   [Deploying apps to KUBE](#deploying-your-app-to-a-kube)
 
+:::showcase[demo=Test#L5-L13]
```

Comment:
i dont think we mean to leave this here right?

Reply:
Correct, this is just a demo for testing purposes. Once this is fully working I'll work with you to get one in a proper stop.